components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -329,18 +329,22 @@ rendering:
     ...
 ```
 
-Now you can use the timespan from the application's code:
+Now you can use the timespan from the application's code. Each time interval that
+the timespan metric records must be associated with an object provided by the user.
+This is so that intervals can be measured concurrently.  For example, to measure page
+load time on a number of tabs that are loading at the same time, each time interval
+should be associated with an object that uniquely represents each tab.
 
 ```Kotlin
-import org.mozilla.yourApplication.GleanMetrics.Rendering
+import org.mozilla.yourApplication.GleanMetrics.Pages
+
+fun onPageStart(e: Event) {
```

Comment:
nit: I would use a different example, this is identical to the timing distribution one and could cause confusion. What about measuring the "startup time"?

Reply:
+1, ideally an example that makes sense to only measure once per app run.
"startup time" doesn't have a "cancel" scenario... maybe something like "logging into the app" or "load app data from disk"?

Reply:
Also update around line 319 and 323 here.

Reply:
Good suggestion.  I'll make the timer example be "logging into the app".